### PR TITLE
fix(toaster): avoid purge animation classes

### PR DESCRIPTION
**Pull request by dine**

### CODE REVIEW

File: src/components/ui-toaster.vue [hunk 1]
```diff
@@ -1,6 +1,10 @@
 <template>
     <div class="fixed top-0 right-0 overflow-x-hidden z-50 max-w-sm">
-        <transition-group tag="div" name="animate-fade-right">
+        <transition-group
+            tag="div"
+            enter-active-class="animate-fade-right-enter-active"
+            leave-active-class="animate-fade-right-leave-active"
+        >
```
Comment (daniarsuprianto) [inline src/components/ui-toaster.vue:7]: ugh

Comment (daniarsuprianto) [reply src/components/ui-toaster.vue:7]: we need to do this for every animation i guess? i read something like this, maybe we could implement that? i mean, to whitelist the vue default animation classes

Comment (daniarsuprianto) [reply src/components/ui-toaster.vue:7]: https://github.com/ky-is/vue-cli-plugin-tailwind/blob/master/generator/templates/postcss.config.js

**Review by daniarsuprianto — APPROVED** [review]

**Review by bartderijk — APPROVED** [review]

File: src/components/ui-toaster.vue (hunk 1 above)
Comment (dine) [reply src/components/ui-toaster.vue:7]: yeah, the only catch of that solution is that if we have a lot of animations they won't be purged at all

**Pull request merged**